presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -387,4 +426,28 @@ public void setNull(PreparedStatement statement, int index)
             }
         };
     }
+
+    private static class DataCompressionDao
+    {
+        private final Handle handle;
+
+        private DataCompressionDao(Handle handle)
+        {
+            this.handle = requireNonNull(handle, "handle is null");
+        }
+
+        Optional<DataCompression> getDataCompression(JdbcTableHandle table)
```

Comment:
Could you please consider different name for this method? 
1. usually get is a lightweight one.
2. currently it repeats the class name. 

Maybe `take` or `takeFor`?

Reply:
I like current name. I don't like suffixes in method like `For`, it adds no value but makes method longer. It is obvious that method will return a value for given parameters.

Reply:
Actually I think `DataCompressionDao` class is not needed, all you need is `getDataCompression` method.

Reply:
I'm fine with prefix `get` as long as it's clearly stated that it's dao, and it is.
With shorter name for method (maybe just `get(JdbcTableHandle table)`) I like this class. If the name left as is static method is enough.